int32 connectionID
int32 pedBicycleDetect
